frontend/components/panels/VaultLiquidationPanel.vue

```diff
@@ -0,0 +1,182 @@
+        },
+        isLiquidating: {
+            type: Boolean,
+            default: false,
+        },
+        walletAddress: {
+            type: String,
+            default: null,
+        },
+        network: {
+            type: String,
+            default: 'mainnet',
+        },
+    },
+    data() {
+        return {
+            isExecuteToAnotherWalletModalShown: false,
+        };
+    },
+    computed: {
+        currentStateAndTitle(): PanelProps {
+            if (!this.isLiquidatable) {
+                return {
+                    name: 'inactive',
+                    title: 'Vault is not ready to be liquidated',
+                };
+            }
+            if (!this.isWalletConnected) {
+                return {
+                    name: 'inactive',
+                    title: 'Vault is ready to be liquidated, but no wallet is connected',
+                };
+            }
+            if (this.areLimitsMissing) {
+                return {
+                    name: 'inactive',
+                    title: 'Vault is ready to be liquidated, but current liquidation limits are not known',
+                };
+            }
+            if (this.areLimitsReached) {
+                return {
+                    name: 'inactive',
+                    title: 'Vault is ready to be liquidated, but current liquidation limits are reached',
+                };
+            }
+            return {
+                name: 'notice',
+                title: 'Vault is ready to be liquidated',
+            };
+        },
+        isLiquidatable(): Boolean {
+            return this.auctionState === 'liquidatable';
+        },
+        debtAndIncentives(): BigNumber {
+            return this.debtDai.plus(this.incentiveRelativeDai).plus(this.incentiveConstantDai);
+        },
+        areLimitsMissing(): boolean {
+            if (!this.liquidationLimits) {
+                return true;
+            }
+            return (
+                Object.values(this.liquidationLimits).includes(null) ||
+                Object.values(this.liquidationLimits).length !== 4
+            );
+        },
+        globalDifference(): BigNumber | undefined {
+            if (this.areLimitsMissing) {
+                return undefined;
+            }
+            return this.liquidationLimits.maximumProtocolDebtDai
+                .minus(this.liquidationLimits.currentProtocolDebtDai)
+                .minus(this.debtAndIncentives);
+        },
+        collateralDifference(): BigNumber | undefined {
+            if (this.areLimitsMissing) {
+                return undefined;
+            }
+            return this.liquidationLimits.maximumCollateralDebtDai
+                .minus(this.liquidationLimits.currentCollateralDebtDai)
+                .minus(this.debtAndIncentives);
+        },
+        areLimitsReached(): Boolean {
+            return this.globalDifference.isNegative() || this.collateralDifference.isNegative();
+        },
```

Comment:
None of those computed props are needed in the panel. One panel should only be responsible for one thing. The limits are checked in another panel, so the code doesn't need to be copied here. Instead, if for some reason execution is not possible, the `disabled=true` prop will be set